README.md

```diff
@@ -1,18 +1,41 @@
-# Truffle-AI-Backend
+# truffle-ai-backend
 
-## Setup
+## Installation
 
-```bash
-git clone git@github.com:jst-seminar-rostlab-tum/truffle-ai-backend.git
-npm install
-npm run prepare
-npm run dev
+Install Docker from https://www.docker.com/ and install dependencies with `npm ci`.<br/>
+Before starting to develop check that you have setup your Environment variables in `.env.dev`! The script `env.sh` checks they are loaded during development.
+
+## Commands
+
+```zsh
+npm ci # install all dependencies from package-lock.json
+
+npm run prepare # installs pre-commit hook (running lint-staged on staged files)
+npm run dev # start services in dev mode
+npm run build # build services
+npm run serve # serve the built services from an endpoint
+
+npm run test # check your files format and lint them
+npm run format # format all your files with Prettier
+npm run lint:fix # lint your files with ESLint and try to fix errors
+
+docker compose build # builds the Docker containers
+docker compose up # run created Docker containers
 ```
 
-`git clone git@github.com:jst-seminar-rostlab-tum/truffle-ai-frontend.git` clones the repository
+## Services
+
+The project consists of two services
+
+- **./packages/graphql_gateway** # Gateway composing multiple GraphQL schemas into one, authentication check on jwt cookies
+- **./packages/graphql_server** # Graphql server
+
+### graphql_gateway
```

Comment:
I would personally remove the graphql_ prefix from the packages everywhere

Reply:
agreed to keep